sdk/eventgrid/azure-resourcemanager-eventgrid/CHANGELOG.md

```diff
@@ -1,7 +1,337 @@
 # Release History
 
-## 1.0.0-beta.2 (Unreleased)
+## 1.0.0-beta.2 (2021-02-18)
 
+- Azure Resource Manager EventGrid client library for Java. This package contains Microsoft Azure SDK for EventGrid Management SDK. Azure EventGrid Management Client. Package tag package-2020-10-preview. For documentation on how to use this package, please see [Azure Management Libraries for Java](https://aka.ms/azsdk/java/mgmt).
+
+### Breaking Change
+
+#### `models.SystemTopic$DefinitionStages` was modified
+
+* Stage 1, 2 was added
+
+#### `models.PartnerNamespace$DefinitionStages` was modified
+
+* Stage 1, 2 was added
+
+#### `models.PartnerRegistration$DefinitionStages` was modified
+
+* Stage 1, 2 was added
+
+#### `models.EventChannel$DefinitionStages` was modified
+
+* Stage 1 was added
```

Comment:
It seems these are New Feature rather than Breaking Change, may need manual change currently.

Reply:
@ChenTanyi .. Yes, we are adding new features in this preview API version